### DISCUSSION TITLE
Is it possible to show EPSG:3395 base map in 3d mode?

### DISCUSSION THREAD

**Discussion by dbezrukov** [TerriaJS/terriajs#5295] [Q&A] [answered]
Hi TerriaJS supporters, 

My TerriaMap instance uses Google and OSM base layers which are in the standard EPSG:3857 projection. 
Each layer is added as UrlTemplateCatalogItem and everything is fine.
I wonder is it possible to add an EPSG:3395 layer served by a tile-server? 
Predictably, when I add an EPSG:3395 layer passing xyz scheme to UrlTemplateCatalogItem, there is a notable offset when switching between EPSG:3395 and OSM.

There's been some movement toward EPSG:3395 support in Cesium:
EPSG:3395 support
https://github.com/CesiumGS/cesium/issues/2084
Support for Proj4JS and Custom Projections in Columbus View and 2D
https://github.com/CesiumGS/cesium/pull/7502
But as I understand it's still just a desirable feature.

Terria uses its own fork of Cesium and has some additional tools for working with layers, - I thought EPSG:3395 could be handled by Terria.

Thank you for the great project!

**Comment by tephenavies** [comment, marked as answer]
Hi @dbezrukov. I'm sorry to say we don't support EPSG:3395 layers and it's not on our list of improvements we want to make for upcoming releases. The best way to get support for it would be to PR it to the official Cesium repo and we can include it in a release from our fork after Cesium release the feature.

**Reply by dbezrukov** [reply]
Thank you, Stephen!
